src/fetch.rs

```diff
@@ -42,7 +42,11 @@ pub fn get_latest_dependency(
         let new_version = if flag_allow_prerelease {
             format!("{}--PRERELEASE_VERSION_TEST", crate_name)
         } else {
-            format!("{}--CURRENT_VERSION_TEST", crate_name)
+            match crate_name {
+                "test_breaking" => "0.2.0".to_string(),
+                "test_nonbreaking" => "0.1.1".to_string(),
```

Comment:
since we're not testing the actual behavior, could you add some unit tests for `old_version_compatible`?